providers/base/units/wireless/jobs.pxu

```diff
@@ -317,6 +317,32 @@ requires:
  {%- endif %}
  net_if_management.device == '{{ interface }}' and net_if_management.managed_by == 'NetworkManager'
 
+unit: template
+template-resource: device
+template-filter: device.category == 'WIRELESS' and device.interface != 'UNKNOWN'
+template-engine: jinja2
+template-unit: job
+id: wireless/wifi7_mlo_{{ interface }}
+template-id: wireless/wifi7_mlo_interface
+depends: wireless/detect
+_summary: Test Wi-Fi 7 Multi-Link-Operation (MLO) on {{ interface }}
+_purpose:
+  Check if the DUT supports Wi-Fi 7 features like Multi-Link-Operation
+plugin: shell
+user: root
```

Comment:
Don't forget to specify the environment variable you want Checkbox to pass to this job:

```suggestion
user: root
environ: MLO_SSID MLO_PSK
```